gapis/memory/blob.go

```diff
@@ -20,6 +20,7 @@ import (
 	"fmt"
 	"io"
 
+	// REMOVE THIS BEFORE CHECKING IN
```

Comment:
Just the comment... don't remove anything else but the comment that says "Remove"

Reply:
Well Remove this comment of course